setup.cfg

```diff
@@ -11,4 +11,15 @@ force_alphabetical_sort = True
 force_single_line = True
 lines_after_imports = 2
 line_length = 200
-not_skip = __init__.py
\ No newline at end of file
+not_skip = __init__.py
+
+
+[flake8]
+ignore = D203
+exclude =
+    parts,
```

Comment:
parts, eggs, bin: this list can get long.
It is better to call flake8 with `flake8 lizard_auth_client` so that it only looks in that subdirectory.

Reply:
Done